src/libstd/sys/redox/ext/unixsocket.rs

```diff
@@ -0,0 +1,254 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![stable(feature = "unix_socket", since = "1.10.0")]
```

Comment:
Stability annotations aren't needed here since these types aren't publicly exposed.

Reply:
Maybe not all, but I believe some of them were needed because even though the module is private the function is public. Could be solved with using pub(crate) instead of pub, but that's ugly